lib/generators/my_zipcode_gem/templates/county_model.rb

```diff
@@ -8,8 +8,16 @@ class County < ActiveRecord::Base
   validates :name, uniqueness: { scope: :state_id, case_sensitive: false },
                    presence: true
 
-  scope :without_zipcodes, -> { joins("LEFT JOIN zipcodes ON zipcodes.county_id = counties.id").where("zipcodes.county_id IS NULL") }
-  scope :without_state, -> { where("state_id IS NULL") }
+  scope :without_zipcodes, -> {
+    county = County.arel_table
+    zipcodes = Zipcode.arel_table
+    zipjoin = county.join(zipcodes, Arel::Nodes::OuterJoin)
+        .on(zipcodes[:county_id].eq(county[:id]))
```

Comment:
I don't remember ever telling hound to review this code :).  Maybe you did which is absolutely fine. Maybe we can update the rules to follow what we want.